.agents/skills/synapse-a2a/SKILL.md

```
@@ -130,19 +153,21 @@ If `[REPLY EXPECTED]` marker is present, you **MUST** reply using `synapse reply
 # When multiple senders are pending, inspect and choose target
-synapse reply --list-targets --from <your_agent_id>
-synapse reply "Here is my analysis..." --from <your_agent_id> --to <sender_id>
+synapse reply --list-targets
+synapse reply "Here is my analysis..." --to <sender_id>
+
+# In sandboxed environments (like Codex), specify your agent ID
+synapse reply "Here is my analysis..." --from <your_agent_id>
 ```
 
 **Example - Question received (MUST reply):**
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)